### PR TITLE
Translate "Ruby 3.4.1 released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translate of #3445

Actual diff is 2719c2a

### CODE REVIEW

File: ko/news/_posts/2024-12-25-ruby-3-4-1-released.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: news_post
+title: "Ruby 3.4.1 Released"
+author: "naruse"
+translator: "shia"
+date: 2024-12-25 10:00:00 +0000
```
Comment (yous) [inline ko/news/_posts/2024-12-25-ruby-3-4-1-released.md:6]: https://github.com/ruby/www.ruby-lang.org/commit/ced8733207718d383eacb62fabddc2a8c448a32b

```suggestion
date: 2024-12-25 00:00:00 +0000
```

File: ko/news/_posts/2024-12-25-ruby-3-4-1-released.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+layout: news_post
+title: "Ruby 3.4.1 Released"
```
Comment (yous) [inline ko/news/_posts/2024-12-25-ruby-3-4-1-released.md:3]:
```suggestion
title: "Ruby 3.4.1 릴리스"
```

File: ko/news/_posts/2024-12-25-ruby-3-4-1-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-12-25-ruby-3-4-1-released.md]: Fixed 9f2e4558d093c5ff5e95d892e0f9b845b137139f

File: ko/news/_posts/2024-12-25-ruby-3-4-1-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2024-12-25-ruby-3-4-1-released.md]: Fixed 9f2e4558d093c5ff5e95d892e0f9b845b137139f

**Review by yous — APPROVED** [review]
LGTM, but let's hold this until #3443 got merged.

**Pull request merged**